src/FlubuCore/Targeting/TargetTree.cs

```diff
@@ -288,33 +290,48 @@ public virtual void LogBuildSummary(IFlubuSession session)
                 }
             }
 
-            int maxTargetNameLength;
-
             if (session.Args.DryRun)
             {
                 session.LogInfo("DRY RUN PERFORMED");
             }
             else if (session.UnknownTarget.HasValue && !session.UnknownTarget.Value)
             {
-                var targets = GetTargetsInExecutionOrder(session);
+                LogBuildSummaryTable(session);
+            }
+        }
+
+        public virtual void LogBuildSummaryTable(IFlubuSession session)
+        {
+            const string TargetTitle = "Target";
+            const string DurationTitle = "Duration";
+            const string StatusTitle = "Status";
+            const int DurationLength = 8;
+            const int StatusLength = 10;
+
+            var targetsInOrder = GetTargetsInExecutionOrder(session);
+            if (targetsInOrder.Count < 1)
+            {
+                return;
+            }
```

Comment:
So when there is no targets specified, build summary is skipped.
For example, when showing a built-in task 'help', it won't show the build summary.